app/data/schema/schema-v1.json

```diff
@@ -17,7 +17,8 @@
                     "enum": [
                         "equals",
                         "not equals",
-                        "contains"
+                        "contains",
+                        "unanswered"
```

Comment:
Not sure about empty as should that include an empty string which it doesn't currently, we should probably not sure empty strings is the fix for that.
For not set we could define it as that but I guess it's just a synonym really and we should choose what our favourite [synonym](http://www.thesaurus.com/browse/unanswered) is